contrib/etime/etime.c

```diff
@@ -121,28 +140,62 @@ stack_push()
 	return stackItem;
 }
 
+static void stack_pop(int64 stackId) {
```

Comment:
The code styles of your 2 commits are not that consistent, of which he former one is closely like ours. Notice you should place the braces of the code block on a single line and use tab instead of spaces always like this (supposing tab take 4 bytes long).

```
if (eTimeEventStack != NULL && eTimeEventStack->stackId == stackId)
{
	MemoryContextDelete(eTimeEventStack->context);
}
else
{
	elog(ERROR,
		"etime stack item " INT64_FORMAT " not found on top - cannot pop",
		stackId);
}
```